src/pint/models/stand_alone_psr_binaries/DDK_model.py

```diff
@@ -161,9 +161,9 @@ def delta_kin_proper_motion(self):
 
         .. math::
 
-            ki = KIN + \delta_{KIN}
+            ki = KIN + \\delta_{KIN}
```

Comment:
This `ki` notation is a little odd, and doesn't really appear in the paper.  Can we make this more clear?